tests/e2e_logging.py

```diff
@@ -17,18 +17,29 @@
 from loguru import logger as LOG
 
 
+def log_get_string(msg):
```

Comment:
Since we do not support schema in Lua, this is necessary :( If there's a quick fix for it, please let me know.

Reply:
We should change the lua imeplementation of the app so it returns the same format as the C++ version, rather than parsing the different formats here.

Reply:
ie - in logging.lua, `LOG_get` should `return { msg = env.get(tables.priv0) }`

Reply:
It would be good to emphasise the equivalence in the documentation as well ("Here are two ways to implement the same application").